spec/test_prof/factory_prof_spec.rb

```diff
@@ -19,38 +19,77 @@
   describe "#result" do
     subject(:result) { described_class.result }
 
-    it "has no stacks when no data created" do
-      FactoryGirl.build_stubbed(:user)
-      User.first
-      expect(result.stacks.size).to eq 0
+    matcher :have_no_stacks do
```

Comment:
I'd prefer no to dry out specs this way; that makes specs less readable (especially when matchers have hard-coded expectations). Sometimes duplication is not evil.